recipes/ginkgo/all/conanfile.py

```diff
@@ -0,0 +1,100 @@
+from conans import ConanFile, tools, CMake
+import os
+
+
+class GinkgoConan(ConanFile):
+    name = "ginkgo"
+    license = "BSD-3-Clause"
+    homepage = "https://github.com/ginkgo-project/ginkgo"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "High-performance linear algebra library for manycore systems, with a focus on sparse solution of linear systems."
+    topics = ("hpc", "linear-algebra")
+    settings = {"os": None,
```

Comment:
Please take a look at our other recipes.

This is a good example https://github.com/conan-io/conan-center-index/blob/master/recipes/zmqpp/all/conanfile.py

Reply:
How can I disable support for certain compilers then? A lot of older compilers tend to crash on our code.

Reply:
Excellent questions, you can copy conan-io/conan#8002